build.gradle

```diff
@@ -12,7 +12,7 @@ subprojects {
       mavenCentral()
     }
     dependencies {
-      classpath 'com.android.tools.build:gradle:1.2.2'
+      classpath 'com.android.tools.build:gradle:1.2.3'
```

Comment:
But we can remove `buildscript` section from android-sample, because it is redundant — subprojects makes the deal.